doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc

```diff
@@ -0,0 +1,102 @@
+= How to install {product} for production ?
+
+This section provides instructions for installing a production-ready version of the application. This setup is intended for deployment in a live environment, with full multi-user access and all necessary security configurations.
+
+== Get {product}
+
+[CAUTION]
+.Stable versions
+====
+Only `YEAR.MONTH.0` versions are stable and suitable for production deployments.
+Other versions, such as intermediate releases, are intended solely for testing purposes.
+They're not recommended for production use due to their potential for further changes and instability.
+Always use the stable `.0` releases for any production environments to ensure reliability and support.
+For more details see xref:user-manual:cycle.adoc[].
+====
+
+== Prepare ecosystem
+
+Pay attention to deploy enterprise grade version of the prerequisites. In particular the databse service. Before do deploy manually the databse, we recomend to use managed database instance (provided by the IT service of a corporation or using cloud services)
+
+[INFO]
+====
+Before you proceed with the installation, please ensure you have the following xref:requirements.adoc[prerequisites]:
+====
+
+We recommand to provide a dedicated host (server or virtual machine) to serve the syson application and only the application.
```

Comment:
recommand -> recommend
to provide -> providing
(server or virtual machine) -> , either a server or a virtual machine,
to serve -> to run
syson -> {product}
and only the application -> exclusively